uber/templates/base.html

```diff
@@ -188,43 +188,6 @@
             return false;
         };
 
-        var getBadgeNumFromBarcode = function(barcode, callback) {
```

Comment:
This was moved to the barcode plugin, which wasn't 100% necessary but I was consolidating the stuff from the panels plugin and figured why not.

Reply:
OMG, thank you for doing this! That's where it belongs, I was just too lazy to open two pull requests in two repos for a single feature.